ext/opcache/tests/jit/bug11917.phpt

```diff
@@ -0,0 +1,73 @@
+--TEST--
+Bug #11917 (primitives seem to be passed via reference instead of by value under some conditions when JIT is enabled on windows)
```

Comment:
Please also move this test to a separate PR. We shouldn't merge a failing test. Somebody would need to find a solution first.